sdk/openai/openai/sources/customizations/api/models.ts

```diff
@@ -105,49 +101,3 @@ export interface AzureExtensionsOptions {
    */
   extensions?: AzureChatExtensionConfiguration[];
 }
-
-/** Information about the content filtering category, if it has been detected. */
-export type ContentFilterResults = ContentFilterSuccessResults | ContentFilterErrorResults;
```

Comment:
Why is this type being deleted?

Reply:
We are using the generated `ContentFilterResults` since now the generated code has the `error` property and the deserialization logic for the error

Reply:
But this type is different and is in fact better because it enables type narrowing when you check the existence of the error field.

Reply:
Hmmm I might have misunderstood from the previous comment. I'll revert the change for this type then.

Reply:
Yeah we can re-used the generated serializer but still have our custom type.